DSharpPlus/Clients/DiscordClient.Dispatch.cs

```diff
@@ -1551,6 +1552,15 @@ internal async Task OnMessageUpdateEventAsync(DiscordMessage message, TransportU
                 message._attachments.AddRange(event_message._attachments);
                 message.Pinned = event_message.Pinned;
                 message.IsTTS = event_message.IsTTS;
+
```

Comment:
Why are you clearing then adding? Couldn't you get away with setting the property, or is it readonly?

Reply:
To keep consistency with the previous code mostly honestly.

Reply:
Fair enough, I suppose